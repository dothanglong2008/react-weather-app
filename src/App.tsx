import styles from "./App.module.scss";
import Weather from "./features/weather/Weather";

function App() {
  return (
    <div className={styles.app}>
      <Weather />
    </div>
  );
}

export default App;
