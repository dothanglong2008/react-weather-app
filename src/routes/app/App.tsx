import Item from "../../components/weather/single-item/item/Item";
import styles from "./App.module.scss";

function App() {
  return (
    <div className={styles.app}>
      <Item />
    </div>
  );
}

export default App;
