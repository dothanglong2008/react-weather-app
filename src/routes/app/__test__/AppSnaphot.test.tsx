import App from "../App";
import renderer from "react-test-renderer";

test("renders Weather component", () => {
  const component = renderer.create(<App />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
