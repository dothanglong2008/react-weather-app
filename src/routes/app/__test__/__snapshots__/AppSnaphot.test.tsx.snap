// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Weather component 1`] = `
<div
  className="app"
/>
`;
